backend/tex/aux_parser.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import json
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--file', help='filepath to input aux')
+args = parser.parse_args()
+
+f = open(args.file, 'r')
+s = '\\newlabel'
```

Comment:
please rename it to PREFIX. Otherwise there is a name clash with `s` in `getArgs`